src/test/scala/symsim/ExperimentSpec.scala

```diff
@@ -17,14 +17,22 @@ trait ExperimentSpec[State, ObservableState, Action]
     outputToFile: Option[String] = None
   ): setup.Policy =
 
-    val (q, r) = setup.runQ
-    val csvString: String = r.mkString("\n")
-    val outputFile: String = "accumulative reward.csv"
-    val writer = new PrintWriter(outputFile)
+    val (q, r, qL) = setup.runQ
+    val csvStrR: String = r.mkString("\n")
+    val outputFileR: String = "accumulative reward.csv"
+    val writerR = new PrintWriter(outputFileR)
     try {
-      writer.println(csvString)
+      writerR.println(csvStrR)
```

Comment:
I think I asked about this before. Why do we use these complex writer things. A normal `print` does not work?

Reply:
I am writing in a file, I did not whether there is a way to write in a file using print!

Reply:
ah ok. Let's leave it as is then. I missed that this is is to a file.